lib/fastlane/helper/git_helper.rb

```diff
@@ -1,9 +1,12 @@
 module Fastlane
   module Actions
-    def self.git_log_between(pretty_format, from, to, include_merges)
+    GIT_MERGE_COMMIT_FILTERING_OPTIONS = [:include_merges, :exclude_merges, :only_include_merges].freeze
```

Comment:
I'm conflicted on moving the filtering options to this level since git itself allows you to pass in both flags if you really want to. I was thinking the git helper would remain as dumb as possible and just map ruby to the command line options and then anything that uses it can turn on and off those options in whatever way they want.